dask/optimize.py

```diff
@@ -45,7 +47,13 @@ def fuse(dsk):
     >>> d = {'a': 1, 'b': (inc, 'a'), 'c': (inc, 'b')}
     >>> fuse(d)  # doctest: +SKIP
     {'c': (inc, (inc, 1))}
+    >>> fuse(d, keys=['b'])  # doctest: +SKIP
+    {'b': (inc, 1), 'c': (inc, 'b')}
     """
+    if isinstance(keys, (set, list)):
+        keys = set(keys)
+    elif keys is not None:
+        keys = set([keys])
```

Comment:
> Sound reasonable?

Yeah, I guess so. I just wasn't sure if there was a need to ever pass in a single key, rather than `[key]`. But I suppose mirroring the interface of get is reasonable.